hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -37,7 +37,11 @@ def local_postgresql?
   end
 
   def local_redis?
-    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy::plugin::pulp', 'pulpcore_enabled')
+    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy_plugin_pulp', 'pulpcore_enabled')
+  end
+
+  def pulpcore_enabled?
+    param_value('foreman_proxy_plugin_pulp', 'pulpcore_enabled')
```

Comment:
Is this correct? I think the class is actually `foreman_proxy::plugin::pulp` as it is in the current code.

Reply:
So I tried this:
mod = Kafo::KafoConfigure.config.modules.find { |m| m.name == "foreman_proxy_plugin_pulp" }
puts mod.params.flatten
and I see all the params correctly. With this foreman_proxy::plugin::pulp I see nil.

Reply:
Sounds like we need tests for these helpers because that's not what I expected.